modules/material/themes/material/mfa/prompt-for-mfa-webauthn.twig

```diff
@@ -37,9 +38,11 @@
       }
 
       function offerRetry() {
-        const retryButton = document.getElementById('retryButton');
+        const verifyBtn = document.getElementById('verifyBtn');
 
-        retryButton.classList.remove('hide');
+        verifyBtn.classList.add('mdl-color-text--red');
+        verifyBtn.classList.remove('mdl-color-text--primary');
+        verifyBtn.textContent = '{{ '{mfa:button_try_again}'|trans }}';
```

Comment:
Do we need to do any explicit escaping in places like this where we're injecting twig values into JavaScript? I don't remember if Twig does that automatically, or if we need to add some other `|js` or something (that's a total guess).